settings.gradle.kts

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
   plugins {
-    id("com.github.ben-manes.versions") version "0.39.0"
+    id("com.github.ben-manes.versions") version "0.41.0"
```

Comment:
I suspect that we aren't using anything from ben-manes any more

Reply:
This isn't caffeine, it's the versions plugin which gives `:dependencyUpdates` task :)